install.sh

```
@@ -0,0 +1,1567 @@
+    fi
+
+    echo -e "${GREEN}✓ offline-repo.yml 준비 완료${NC}"
+    echo ""
+}
+
+# (삭제됨: check_kubespray_ready - prepare_kubespray_source와 prepare_offline_resources로 대체)
+
+# ansible 설치 확인 및 설정
+check_or_setup_ansible() {
+    local kubespray_dir="${SCRIPT_DIR}/kubespray-${VERSION}"
+    local requirements_txt="${kubespray_dir}/requirements.txt"
```

Shellcheck (0.11.0)

[warning] requirements_txt appears unused. Verify use (or export if used externally).

(SC2034)